app/assets/images/payment_icons/swiftpay.svg

```diff
@@ -0,0 +1 @@
45.645a1.208 1.208 0 01-.373.423 1.673 1.673 0 01-.537.234c-.202.046-.41.07-.626.07a3.47 3.47 0 01-.505-.038 4.001 4.001 0 01-.506-.108 4.662 4.662 0 01-.48-.177 2.73 2.73 0 01-.424-.233l.455-.904a.865.865 0 00.165.12c.093.059.206.118.341.177.14.059.293.111.461.158.17.046.34.07.512.07.367 0 .55-.112.55-.336a.283.283 0 00-.082-.208.726.726 0 00-.227-.146 2.182 2.182 0 00-.348-.126l-.43-.127a4.016 4.016 0 01-.537-.202 1.514 1.514 0 01-.38-.259.988.988 0 01-.22-.335 1.282 1.282 0 01-.07-.442c0-.236.044-.445.133-.626.088-.181.208-.333.36-.455.152-.126.326-.221.525-.284.202-.064.414-.095.638-.095.156 0 .31.015.461.044.152.03.297.067.436.114.14.046.268.099.386.158.122.059.234.118.335.177l-.455.86zm5.216-.146h.96l-1.37 3.318h-.829l-.511-1.365-.513 1.365H13.7l-1.372-3.318h.955l.904 2.383.31-.904-.589-1.479h.81l.322.999.316-.999h.809l-.582 1.479.31.904.897-2.383zm1.4 3.318v-3.318h1.01v3.318h-1.01zm0-3.697V9.27h1.01v.917h-1.01zm2.036 3.697v-2.282h-.423v-.764h.423v-.272c0-.207.028-.394.082-.563.055-.168.133-.311.234-.43.101-.117.223-.208.367-.27.143-.064.301-.096.474-.096.27 0 .558.066.866.196l-.184.759a1.101 1.101 0 00-.227-.07.884.884 0 00-.228-.032c-.248 0-.373.156-.373.468v.31h.772v.764h-.772v2.282h-1.011zm4.484-.177c-.139.06-.299.114-.48.164-.177.051-.356.076-.537.076-.127 0-.247-.017-.36-.05a.792.792 0 01-.291-.146.763.763 0 01-.202-.278 1.087 1.087 0 01-.07-.41v-1.732h-.424v-.765h.424V9.511h1.011v1.056h.677v.765h-.677v1.384c0 .11.028.19.083.24.054.046.124.07.208.07a.78.78 0 00.234-.038c.08-.026.15-.053.208-.082l.196.802zm.595.177V9.398h1.915c.215 0 .41.044.588.132a1.572 1.572 0 01.765.834c.075.182.113.365.113.55 0 .199-.035.388-.107.57a1.536 1.536 0 01-.291.486 1.339 1.339 0 01-1.036.468h-.91v1.447h-1.037zm1.036-2.351h.848a.401.401 0 00.315-.158c.09-.105.133-.26.133-.462a.747.747 0 00-.044-.271.525.525 0 00-.108-.19.39.39 0 00-.157-.107.445.445 0 00-.178-.038h-.809v1.226zm3.85 2.414c-.165 0-.319-.025-.463-.076a1.154 1.154 0 01-.36-.22 1.022 1.022 0 01-.329-.759c0-.165.034-.314.102-.449.072-.135.17-.25.297-.348.126-.096.276-.172.449-.227a2.112 2.112 0 01.98-.044c.13.02.244.054.34.1v-.15c0-.384-.22-.576-.663-.576-.181 0-.354.032-.518.095-.165.059-.338.15-.518.272l-.298-.633c.22-.143.447-.25.683-.322.236-.072.487-.107.752-.107.501 0 .89.118 1.163.354.274.235.41.581.41 1.036v.853c0 .102.016.173.045.215.034.038.09.061.17.07v.853a1.45 1.45 0 01-.24.038 1.736 1.736 0 01-.196.012c-.19 0-.332-.035-.43-.107a.502.502 0 01-.182-.316l-.02-.152a1.504 1.504 0 01-1.175.588zm.296-.727a.895.895 0 00.575-.208c.097-.076.145-.156.145-.24v-.316a1.466 1.466 0 00-.29-.076 1.384 1.384 0 00-.285-.032.795.795 0 00-.46.127.387.387 0 00-.178.328c0 .122.046.224.139.304a.54.54 0 00.354.113zm2.528 1.163a1.235 1.235 0 00.436.082.342.342 0 00.284-.126c.068-.084.129-.236.184-.455l-1.264-3.318h1.043l.77 2.452.664-2.452h.948l-1.264 3.855a1.301 1.301 0 01-.474.664 1.3 1.3 0 01-.809.259c-.084 0-.17-.008-.259-.025a1.261 1.261 0 01-.259-.07v-.866z" fill="#3f3e3e" fill-rule="nonzero"/></svg>
```

Comment:
We just need to remove `fill-rule="evenodd" clip-rule="evenodd" stroke-linejoin="round" stroke-miterlimit="2"` at the `<svg>` level and the rest is good.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="38" height="24" viewBox="0 0 38 24" role="img" aria-labelledby="pi-swiftpay"><title id="pi-swiftpay">SwiftPay</title><path d="M35 0H3C1.3 0 0 1.3 0 3v18c0 1.7 1.4 3 3 3h32c1.7 0 3-1.3 3-3V3c0-1.7-1.4-3-3-3z" fill-opacity=".07" fill-rule="nonzero"/><path d="M35 1c1.1 0 2 .9 2 2v18c0 1.1-.9 2-2 2H3c-1.1 0-2-.9-2-2V3c0-1.1.9-2 2-2h32" fill="#fff" fill-rule="nonzero"/><path d="M4.86 9.245c0 .326-.283.59-.63.59-.348 0-.63-.264-.63-.59 0-.326.282-.59.63-.59.347 0 .63.264.63.59zm0 5.51c0 .326-.283.59-.63.59-.348 0-.63-.264-.63-.59 0-.326.282-.59.63-.59.347 0 .63.264.63.59zm-2.31-.59c.348 0 .63-.265.63-.591 0-.326-.282-.59-.63-.59-.347 0-.629.264-.629.59 0 .326.282.59.63.59zm.63-3.74c0 .327-.282.591-.63.591-.347 0-.629-.264-.629-.59 0-.326.282-.59.63-.59.347 0 .63.264.63.59zm2.728 3.74c.349 0 .63-.265.63-.591 0-.326-.281-.59-.63-.59-.347 0-.63.264-.63.59 0 .326.283.59.63.59zM4.86 12c0 .326-.282.59-.63.59-.347 0-.629-.264-.629-.59 0-.326.282-.59.63-.59.347 0 .63.264.63.59zm1.05-.984c.348 0 .63-.264.63-.59 0-.326-.282-.59-.63-.59-.348 0-.63.264-.63.59 0 .326.282.59.63.59z"/><path d="M11.573 10.713a.616.616 0 00-.133-.095 2.78 2.78 0 00-.657-.278 1.494 1.494 0 00-.405-.057c-.37 0-.556.124-.556.372 0 .076.019.14.057.19.042.05.101.097.177.139.08.038.18.074.297.108l.41.113c.216.06.41.125.582.196.173.068.318.154.436.26a.976.976 0 01.278.372c.068.148.101.325.101.531 0 .253-.048.468-.145.645a1.208 1.208 0 01-.373.423 1.673 1.673 0 01-.537.234c-.202.046-.41.07-.626.07a3.47 3.47 0 01-.505-.038 4.001 4.001 0 01-.506-.108 4.662 4.662 0 01-.48-.177 2.73 2.73 0 01-.424-.233l.455-.904a.865.865 0 00.165.12c.093.059.206.118.341.177.14.059.293.111.461.158.17.046.34.07.512.07.367 0 .55-.112.55-.336a.283.283 0 00-.082-.208.726.726 0 00-.227-.146 2.182 2.182 0 00-.348-.126l-.43-.127a4.016 4.016 0 01-.537-.202 1.514 1.514 0 01-.38-.259.988.988 0 01-.22-.335 1.282 1.282 0 01-.07-.442c0-.236.044-.445.133-.626.088-.181.208-.333.36-.455.152-.126.326-.221.525-.284.202-.064.414-.095.638-.095.156 0 .31.015.461.044.152.03.297.067.436.114.14.046.268.099.386.158.122.059.234.118.335.177l-.455.86zm5.216-.146h.96l-1.37 3.318h-.829l-.511-1.365-.513 1.365H13.7l-1.372-3.318h.955l.904 2.383.31-.904-.589-1.479h.81l.322.999.316-.999h.809l-.582 1.479.31.904.897-2.383zm1.4 3.318v-3.318h1.01v3.318h-1.01zm0-3.697V9.27h1.01v.917h-1.01zm2.036 3.697v-2.282h-.423v-.764h.423v-.272c0-.207.028-.394.082-.563.055-.168.133-.311.234-.43.101-.117.223-.208.367-.27.143-.064.301-.096.474-.096.27 0 .558.066.866.196l-.184.759a1.101 1.101 0 00-.227-.07.884.884 0 00-.228-.032c-.248 0-.373.156-.373.468v.31h.772v.764h-.772v2.282h-1.011zm4.484-.177c-.139.06-.299.114-.48.164-.177.051-.356.076-.537.076-.127 0-.247-.017-.36-.05a.792.792 0 01-.291-.146.763.763 0 01-.202-.278 1.087 1.087 0 01-.07-.41v-1.732h-.424v-.765h.424V9.511h1.011v1.056h.677v.765h-.677v1.384c0 .11.028.19.083.24.054.046.124.07.208.07a.78.78 0 00.234-.038c.08-.026.15-.053.208-.082l.196.802zm.595.177V9.398h1.915c.215 0 .41.044.588.132a1.572 1.572 0 01.765.834c.075.182.113.365.113.55 0 .199-.035.388-.107.57a1.536 1.536 0 01-.291.486 1.339 1.339 0 01-1.036.468h-.91v1.447h-1.037zm1.036-2.351h.848a.401.401 0 00.315-.158c.09-.105.133-.26.133-.462a.747.747 0 00-.044-.271.525.525 0 00-.108-.19.39.39 0 00-.157-.107.445.445 0 00-.178-.038h-.809v1.226zm3.85 2.414c-.165 0-.319-.025-.463-.076a1.154 1.154 0 01-.36-.22 1.022 1.022 0 01-.329-.759c0-.165.034-.314.102-.449.072-.135.17-.25.297-.348.126-.096.276-.172.449-.227a2.112 2.112 0 01.98-.044c.13.02.244.054.34.1v-.15c0-.384-.22-.576-.663-.576-.181 0-.354.032-.518.095-.165.059-.338.15-.518.272l-.298-.633c.22-.143.447-.25.683-.322.236-.072.487-.107.752-.107.501 0 .89.118 1.163.354.274.235.41.581.41 1.036v.853c0 .102.016.173.045.215.034.038.09.061.17.07v.853a1.45 1.45 0 01-.24.038 1.736 1.736 0 01-.196.012c-.19 0-.332-.035-.43-.107a.502.502 0 01-.182-.316l-.02-.152a1.504 1.504 0 01-1.175.588zm.296-.727a.895.895 0 00.575-.208c.097-.076.145-.156.145-.24v-.316a1.466 1.466 0 00-.29-.076 1.384 1.384 0 00-.285-.032.795.795 0 00-.46.127.387.387 0 00-.178.328c0 .122.046.224.139.304a.54.54 0 00.354.113zm2.528 1.163a1.235 1.235 0 00.436.082.342.342 0 00.284-.126c.068-.084.129-.236.184-.455l-1.264-3.318h1.043l.77 2.452.664-2.452h.948l-1.264 3.855a1.301 1.301 0 01-.474.664 1.3 1.3 0 01-.809.259c-.084 0-.17-.008-.259-.025a1.261 1.261 0 01-.259-.07v-.866z" fill="#3f3e3e" fill-rule="nonzero"/></svg>
```

Reply:
@tauthomas01 Done